Jamulus.pro

```diff
@@ -88,6 +88,14 @@ win32 {
             advapi32.lib \
             winmm.lib \
             ws2_32.lib
+        greaterThan(QT_MAJOR_VERSION, 5) {
+            # Qt5 had a special qtmain library which took care of forwarding the MSVC default WinMain() entrypoint to
+            # the platform-agnostic main().
+            # Qt6 is still supposed to have that lib under the new name QtEntryPoint. As it does not seem
+            # to be effective when building with qmake, we are rather instructing MSVC to use the platform-agnostic
+            # main() entrypoint directly:
+            QMAKE_LFLAGS += /subsystem:windows /ENTRY:mainCRTStartup
+        }
```

Comment:
@ann0see 
> Could you please add a comment to this what this does Linker Flags? Maybe move it elsewhere down if possible

Github makes it hard to reply to your [previous thread](https://github.com/jamulussoftware/jamulus/pull/2299#discussion_r849874437) as the code has been changed/moved, therefore replying here:

I've done further tests and research regarding this. The above comment basically explains all the reasoning. I also agree with you regarding placement: main.cpp might be a logical place as the build flags are about the main function, but it's rather surprising to find linker flag adjustments in a .cpp file. Therefore, I have moved this to Jamulus.pro where other linker flags are set.